Fixes AttributeError in terrain-based position command (#191)
NVIDIA-Omniverse/orbit: source/extensions/omni.isaac.orbit/docs/CHANGELOG.rst

Changelog
---------
0.10.11 (2024-01-08)
~~~~~~~~~~~~~~~~~~~~

Fixed
^^^^^

> * Fixed AttributeError in :class:`omni.isaac.orbit.env.TerrainBasedPositionCommand.` and added dummy function that returns
>   env_origins as terrain based targets in :class:`omni.isaac.orbit.terrain.TerrainImporter`.

Reviewer: ```suggestion
* Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` command term.
* Added a dummy function in :class:`omni.isaac.orbit.terrain.TerrainImporter` that returns environment origins as terrain-aware sampled targets. This function should be implemented by child classes based on the terrain type.
```

> * Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` command term.
> * Added a dummy function in :class:`omni.isaac.orbit.terrain.TerrainImporter` that returns environment origins as terrain-aware sampled targets. This function should be implemented by child classes based on the terrain type.

Reviewer: ```suggestion
* Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` 
   command term.
* Added a dummy function in :class:`omni.isaac.orbit.terrain.TerrainImporter` that returns environment
   origins as terrain-aware sampled targets. This function should be implemented by child classes based on
   the terrain type.
```

Changelog
---------

> 0.10.11 (2024-01-08)
> ~~~~~~~~~~~~~~~~~~~~
> 
> Fixed
> ^^^^^
> 
> * Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` 
>    command term.
> * Added a dummy function in :class:`omni.isaac.orbit.terrain.TerrainImporter` that returns environment
>    origins as terrain-aware sampled targets. This function should be implemented by child classes based on
>    the terrain type.
> 

Reviewer: ```suggestion
0.10.11 (2024-01-08)
~~~~~~~~~~~~~~~~~~~~

Fixed
^^^^^

* Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` 
  command term.
* Added a dummy function in :class:`omni.isaac.orbit.terrain.TerrainImporter` that returns environment
  origins as terrain-aware sampled targets. This function should be implemented by child classes based on
  the terrain type.

```

Changelog
---------
0.10.11 (2024-01-08)
~~~~~~~~~~~~~~~~~~~~

Fixed
^^^^^

> * Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand` 
>   command term.

Reviewer: ```suggestion
* Fixed attribute error raised when calling the :class:`omni.isaac.orbit.envs.mdp.TerrainBasedPositionCommand`
  command term.
```
